### PR TITLE
Update: magic-pod-api-client

### PR DESCRIPTION
**Pull request by mwakizaka**
Could you check in terms of `go.sum` file which I'm not familiar with.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,6 +1,10 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/Magic-Pod/magic-pod-api-client v0.0.0-20200517225049-a961321ddbcc h1:EoB9osNZpBx5U7Rc0AohsAerC1UKC3GH0bsn9ypJHBg=
 github.com/Magic-Pod/magic-pod-api-client v0.0.0-20200517225049-a961321ddbcc/go.mod h1:BUi+TOp4CBcVNFGfS+XzmoO/az+YGlPJ41An2c5Xr/A=
+github.com/Magic-Pod/magic-pod-api-client v0.0.0-20201101085056-07a201130634 h1:q0n3eprHrP0EbZRZG2iNkaSABOXPZB4bzJOwF0ujRfo=
+github.com/Magic-Pod/magic-pod-api-client v0.0.0-20201101085056-07a201130634/go.mod h1:BUi+TOp4CBcVNFGfS+XzmoO/az+YGlPJ41An2c5Xr/A=
```
Comment (nkns165) [inline go.sum]: I think these four lines are not necessary any more.  It seems already added dependencies are not automatically removed. (https://horizoon.jp/post/2019/04/18/contributing_with_gomodules/ Chapter 5)

But when I tried `go mod tidy`, a new module was added to `go.mod` and `go.sum`.  So manually removing these lines will be enough.  If `bitrise run test` works, it's ok.

**Review by nkns165 — CHANGES_REQUESTED** [review]
Reviewed.

File: go.sum (hunk 1 above)
Comment (mwakizaka) [reply go.sum]: Thanks. I dropped the old ones.

**Review by nkns165 — APPROVED** [review]
👍

**Pull request merged**